.github/workflows/project.yml

```diff
@@ -1,24 +1,80 @@
-# This workflow will automatically add an issue or pull request to the default project.
-# The project needs to be updated when we move to a new default project.
-# https://github.com/marketplace/actions/assign-to-one-project
+# This workflow will automatically assign issues and pull requests to the new default project and relocate issues to the project based on changes in their labels.
+# https://github.com/actions/add-to-project
 
 name: assign-project
 
 on:
   issues:
-    types: [opened, labeled]
+    types:
+      - opened
+      - labeled
   pull_request:
-    types: [opened, labeled]
+    types:
+      - opened
+      - labeled
+
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
-  assign-one-project:
-    name: Assign to one default project
+  add-issue-to-project:
+    name: Assign NEW issue to project with To do status
```

Comment:
Better to enclose those column names with "" so it is clearer that they are column names, as you did at the bottom. For all.